lib/ansible/plugins/netconf/__init__.py

```diff
@@ -32,7 +32,10 @@
     from ncclient.xml_ import to_xml, to_ele, NCElement
     HAS_NCCLIENT = True
     NCCLIENT_IMP_ERR = None
-except (ImportError, AttributeError) as err:  # paramiko and gssapi are incompatible and raise AttributeError not ImportError
+# paramiko and gssapi are incompatible and raise AttributeError not ImportError
+# When running in FIPS mode, cryptography raises InternalError
+# https://bugzilla.redhat.com/show_bug.cgi?id=1778939
+except Exception as err:
```

Comment:
Can't you catch `cryptography.exceptions.InternalError`?

Reply:
@webknjaz I wanted to do that too, but `cryptography` may not always be available. https://github.com/ansible/ansible/pull/65477#discussion_r353775154

Reply:
Well, that could be made conditional, right?
```python
try:
    from cryptography.exceptions import InternalError
except ImportError:
    fips_excs = ()
else:
    fips_excs = (InternalError, )
```
and then
```python
try:
    import ncclient...
except (ImportError, AttributeError, *fips_excs) as err:
    ...
```

Reply:
By the way, FTR cryptography runs a FIPS job in their CI since v3.0: https://github.com/pyca/cryptography/commit/4a245a657a071dff959d30eff1d214c7ff88eada / https://github.com/pyca/cryptography/pull/5323.

Reply:
@webknjaz this is 3rd time they change the error emitted, at this point to deal with FIPS it seems `Exception` is only way forward, unless we want to be back here in 2-3 yrs getting the 'new exception' added to the list.